Source/Core/VideoBackends/OGL/Render.cpp

```diff
@@ -711,10 +712,13 @@ Renderer::Renderer()
   // options while running
   g_Config.bRunning = true;
 
-  glStencilFunc(GL_ALWAYS, 0, 0);
-  glBlendFunc(GL_ONE, GL_ONE);
+  // The stencil is used for bounding box emulation when SSBOs are not available
+  glDisable(GL_STENCIL_TEST);
+  glStencilFunc(GL_ALWAYS, 1, 0);
+  glStencilOp(GL_KEEP, GL_INCR, GL_INCR);
```

Comment:
I'd prefer KEEP, KEEP, REPLACE. INCR has to read the old stencil value and so might be slower.

To be honest, I'm not sure about the second argument. No clue if the GC updates it on shading/blending/whatever...